packages/payment-detection/src/thegraph/queries/GetPaymentsAndEscrowState.graphql

```diff
@@ -28,5 +30,7 @@ query GetPaymentsAndEscrowState(
     from
     timestamp
     block
+    gasUsed
+    gasPrice
```

Comment:
These fields are not available for escrow. I'm not sure it makes sense to add them 🤔

Reply:
actually it's available on escrowEvents. but my question is, does the escrowEvents don't need those fields? if yes, maybe I made a mistake on my change on the subgraph repo...